client/MentorPanel/src/components/scrolling_questions.js

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import PropTypes from 'prop-types'
+import classNames from 'classnames'
+import { Button, ListItem } from '@material-ui/core'
+import { withStyles } from '@material-ui/core/styles'
+
+import { normalizeString } from 'src/funcs/funcs'
+
+class ScrollingQuestions extends React.Component {
```

Comment:
does this have to be `React.Component`? Is because it uses `document`?

Reply:
Because it needs componentDidUpdate()

Reply:
react hook `useEffect` can usually be used in place of `React.Component::componentDidUpdate`

https://reactjs.org/docs/hooks-effect.html

We should make a concerted effort to stick to the functional-component style of React. Mainly because it's better to not have to support two distinct styles of React